.github/workflows/build.yml

```diff
@@ -58,6 +58,13 @@ jobs:
           docker build --load -t ${DOCKER_IMAGE_PREFIX:-dirigiblelabs/}xsk --build-arg JDK_TYPE=external-jdk --build-arg JDK_HOME=jdk .
           docker push ${DOCKER_IMAGE_PREFIX:-dirigiblelabs/}xsk
           cd ../../
+      - name: Push XSK Runtime
+        run: |
+          cd releng/server-runtime
+          cp -r "${JAVA_HOME}" jdk
+          docker build --load -t ${DOCKER_IMAGE_PREFIX:-dirigiblelabs/}xsk-runtime --build-arg JDK_TYPE=external-jdk --build-arg JDK_HOME=jdk .
+          docker push ${DOCKER_IMAGE_PREFIX:-dirigiblelabs/}xsk-runtime
```

Comment:
For this step we will need new repository in dirigiblelabs  `dirigiblelabs/xsk-runtime`